cinder/volume/manager.py

```diff
@@ -680,6 +680,43 @@ def _set_resource_host(self, resource):
             resource.host = volume_utils.append_host(self.host, pool)
             resource.save()
 
+    def _propogate_volume_scheduler_hints(self, context, volume):
+        """Ensure metadata hints are propagated to referenced volumes."""
+        hints = volume_utils.get_scheduler_hints_from_volume(volume)
+        if hints:
+            LOG.debug("Found hints for %(volume)s - %(hints)s",
+                      {'volume': volume.id, 'hints': hints})
+            for hint_key in hints.keys():
+                for vol_id in hints[hint_key]:
+                    try:
+                        meta_vol = objects.Volume.get_by_id(context, vol_id)
+                        LOG.debug("Looking for hints in %s", meta_vol.id)
+                        # Because pep8 error otherwise with lines 696 and 710
+                        vut = volume_utils
+                        subhints = vut.get_scheduler_hints_from_volume(
```

Comment:
So ... `subhints` are the hints already existing on the `meta_vol`. Imho better called `meta_vol_hints` or `meta_hints`.

Reply:
done